### DISCUSSION TITLE
Are there CSB benefits for students?

### DISCUSSION THREAD

**Discussion by Lathryx** [codesandbox/codesandbox-client#5631] [General]
I'm currently a student, and I have the GitHub Student Developer Package. I was wondering if there were any _CodeSandbox_ benefits for students or members of the GitHub Student Developer Package. 

Is there, or, will there be any in the future?

**Comment by guptaashwanee** [comment]
Hey, @Lathryx did you able to get any CodeSandbox benefits.

**Comment by filipelima18** [comment]
Hey @Lathryx. CodeSandbox now offers special plans for education, including a 25% discount on CodeSandbox Pro for students. More details in the [Docs](https://codesandbox.io/docs/learn/plans/education-plans).
